src/zenml/zen_stores/migrations/utils.py

```diff
@@ -236,9 +236,17 @@ def backup_database_to_storage(
                 # correct order, since some tables have inner foreign key
                 # constraints.
                 if "created" in table.columns:
-                    order_by = table.columns["created"]
+                    order_by = [table.columns["created"]]
                 else:
-                    order_by = None
+                    order_by = []
+                if "id" in table.columns:
```

Comment:
do we currently have any entitiy without id? In that case what would we use as tiebreak?

Reply:
All entities that have a `create` field also have an `id` field. This is guaranteed by the fact that they both come from the `BaseSchema` class.

As for entities that don't have either, we don't need a tiebreak. The implicit ordering supported by the storage engine (e.g. insert order or primary key order) is used. This implicit ordering does not create duplicates.